backend/__tests__/routes/doctors.test.js

```diff
@@ -0,0 +1,24 @@
+const request = require("supertest");
+const app = require("../../index");
+const db = require("../../config/database");
+
+beforeAll(() => {
+    // test DB
+    db.authenticate();
+});
+afterAll(() => {
+    // Closing the DB connection allows Jest to exit successfully.
+    db.close();
+});
+
+describe("PATCH: update the list of details that the patient has to provide", () => {
+    it("Returns code 400 if patient ID is not found", () => {
+        const data = {
+            weight: true,
+            temperature: true,
+            symptoms: true,
+            patientId: 100000,
+        };
+        return request(app).patch("/doctors/updateRequiredDetails").send(data).expect(400);
```

Comment:
This test case will never work because the query will always return "successful".
See here:
![image](https://user-images.githubusercontent.com/43119712/154831932-9e4782d7-f71c-4675-9eb1-966d30590a8c.png)

You would need to check if the patient exists before attempting to update if you want the behavior outlined in this test to happen.

Reply:
wrong PR, mb